.svglintrc.mjs

```diff
@@ -1,11 +1,26 @@
-const fs = require('fs');
-
-const data = require('./_data/simple-icons.json');
-const { htmlFriendlyToTitle } = require('./scripts/html-friendly-to-title.cjs');
-const htmlNamedEntities = require('named-html-entities-json');
-const svgpath = require('svgpath');
-const svgPathBbox = require('svg-path-bbox');
-const parsePath = require('svg-path-segments');
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { htmlFriendlyToTitle } from './scripts/utils.js';
+import svgpath from 'svgpath';
+import svgPathBbox from 'svg-path-bbox';
+import parsePath from 'svg-path-segments';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const dataFile = path.join(__dirname, '_data', 'simple-icons.json');
+const htmlNamedEntitiesFile = path.join(
+  __dirname,
+  'node_modules',
+  'named-html-entities-json',
+  'index.json',
+);
+const svglintIgnoredFile = path.join(__dirname, '.svglint-ignored.json');
+
+const data = JSON.parse(fs.readFileSync(dataFile, 'utf8'));
+const htmlNamedEntities = JSON.parse(
+  fs.readFileSync(htmlNamedEntitiesFile, 'utf8'),
```

Comment:
Would you mind to change these sync function to async?

```js
import {promises as fs} from 'node:fs';

const dataFile = JSON.parse(await fs.readFile(svglintIgnoreFile));
```

Reply:
I didn't change it because top level await requires users to be on Node v14

Reply:
Oh, I see.